documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc

```diff
@@ -3329,6 +3329,12 @@ int removed = session.createNativeQuery( removeQuery ).executeUpdate();
 ----
 ====
 
+[WARNING]
+====
+Support for the `$regexp` operator is limited to the string syntax. We do not support the `/pattern/` syntax as it is not
```

Comment:
Personally, I think an example would be much more effective. Are there any limitation in using one syntax compare to the other?